devstack/lib/nfp

```diff
@@ -117,9 +116,6 @@ function create_nfp_gbp_resources {
         gbp service-profile-create --servicetype LOADBALANCERV2 --insertion-mode l3 --shared True --service-flavor service_vendor=haproxy_lbaasv2,device_type=nova,flavor=m1.small --vendor NFP lbv2_profile
         gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP vyos_fw_profile
         gbp service-profile-create --servicetype VPN --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP vpn_profile
-        if [[ $NFP_DEVSTACK_MODE = enterprise ]]; then
-            gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=asav,device_type=nova --vendor NFP asav_fw_profile
-        fi
         create_ext_net
         create_ep_and_nsp
         create_advance_sharing_ptg
```

Comment:
Is this needed as sharing feature will not be submitted? Please check with @ashutosh-mishra